src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -604,9 +670,18 @@ struct cudnn_convolution_impl_fwd_t : public cudnn_convolution_impl_base_t {
                 = utils::downcast<sycl_cuda_stream_t *>(service_stream);
         auto handle = cuda_stream->get_cudnn_handle();
 
-        CHECK(CUDNN_EXECUTE_FUNC_S(cudnnGetConvolutionForwardWorkspaceSize,
-                handle, descs[x], weights_desc, conv_desc, descs[y],
-                fwd_alg_kind, &scratchpad_size));
+        if (data_types[y] == CUDNN_DATA_INT8 && do_scaling) {
```

Comment:
`do_scaling` is a variable that is `true` if any of the `attr-scales` is used. Is there any way to know on this function if the scaling is applied just to the destination ?

Reply:
It can be obtained from pd attributes: 
`do_dst_scaling = !pd()->attr()->scales_.get(DNNL_ARG_DST).has_default_values();`
It could be done in `configure_parameters` right after `do_scaling`:
https://github.com/oneapi-src/oneDNN/blob/274ee103c0c45dbb8aa255203d3d8627dda7ed3f/src/gpu/nvidia/cudnn_convolution_impl.hpp#L136